docs/source/quantum_networks/network_ansatzes.rst

```diff
@@ -4,10 +4,11 @@ Network Ansatzes
 .. currentmodule:: qnetvo
 
 The quantum network ansatz is a collection of network nodes aggregated
-into three sequential layers: prepare, noise, and measure.
+into sequential layers ordered as prepare nodes, noise or processing nodes, and measure nodes.
+It is important that all measurement nodes are in the final layer.
```

Comment:
Not sure if we'll need to reword a little once LOCC is supported - i.e., all "output" measurements must be in the final layer, but you could have intermediate measurements as part of LOCC.